addOns/ascanrulesBeta/src/main/java/org/zaproxy/zap/extension/ascanrulesBeta/HttpParameterPollutionScanRule.java

```diff
@@ -38,17 +39,26 @@
 import org.parosproxy.paros.network.HtmlParameter;
 import org.parosproxy.paros.network.HttpMessage;
 import org.zaproxy.addon.commonlib.CommonAlertTag;
+import org.zaproxy.addon.commonlib.PolicyTag;
 
 /**
  * TODO note that this should extend AbstractAppParamPlugin rather than find parameters internally
  */
 public class HttpParameterPollutionScanRule extends AbstractAppPlugin
         implements CommonActiveScanRuleInfo {
-    private static final Map<String, String> ALERT_TAGS =
-            CommonAlertTag.toMap(
-                    CommonAlertTag.OWASP_2021_A03_INJECTION,
-                    CommonAlertTag.OWASP_2017_A01_INJECTION,
-                    CommonAlertTag.WSTG_V42_INPV_04_PARAM_POLLUTION);
+    private static final Map<String, String> ALERT_TAGS;
+
+    static {
+        Map<String, String> alertTags =
+                new HashMap<>(
+                        CommonAlertTag.toMap(
+                                CommonAlertTag.OWASP_2021_A03_INJECTION,
+                                CommonAlertTag.OWASP_2017_A01_INJECTION,
+                                CommonAlertTag.WSTG_V42_INPV_04_PARAM_POLLUTION));
+        alertTags.put(PolicyTag.QA_FULL.getTag(), "");
```

Comment:
Was this one on purpose? It was not being included in any policy before (spreadsheet still has it excluded from QAs).

Reply:
Nope mustard been a bad c/p